services/order.py

```diff
@@ -0,0 +1,40 @@
+from django.utils import timezone
+from db.models import User, Ticket, Order, MovieSession
+from typing import Optional
+from django.db import transaction
+
+
+def create_order(*, username, tickets, date=None):
+
+    with transaction.atomic():
+        user = User.objects.get(username=username)
+
+        # Якщо дата не передана, використовуємо поточний час
+        if not date:
+            date = timezone.now()
+
+        # Створення замовлення
+        order = Order.objects.create(user=user, created_at=date)
+
+        # Створення квитків для цього замовлення
+        for ticket in tickets:
+            movie_session = MovieSession.objects.get(id=ticket['movie_session'])
```

Comment:
Using `get()` without handling exceptions can lead to `DoesNotExist` errors if the `MovieSession` is not found. Consider using `get_object_or_404()` or handling the exception to provide a more user-friendly error message.